emcc.py

```diff
@@ -2866,10 +2866,10 @@ def get_full_import_name(name):
 
   # check if we can address the 2GB mark and higher: either if we start at
   # 2GB, or if we allow growth to either any amount or to 2GB or more.
-  if settings.INITIAL_MEMORY > 2 * 1024 * 1024 * 1024 or \
+  if not settings.MEMORY64 and (settings.INITIAL_MEMORY > 2 * 1024 * 1024 * 1024 or
```

Comment:
Why is memory64 excluded here? Because pointers are bigints there anyhow? Worth a comment if so I think.

Reply:
Yes, pointers in MEMORY64 all come in either as BigInt or `int32`.  In neither case is `>>>0` ever needed.. and in fact is will break points larger then 4GB.

Reply:
maybe we should rename `CAN_ADDRESS_2GB` but I'll leave a comment for now.

Reply:
Added comment looks good, thanks. Actually though, maybe move it to `settings.js` on `CAN_ADDRESS_2GB`?

Reply:
Done.

Reply:
BTW this is an internal setting (not directly settable)